module/userProjects/helper.js

```diff
@@ -3894,6 +3995,21 @@ function _projectTask(tasks, isImportedFromLibrary = false, parentTaskId = '') {
 	return tasks
 }
 
+function _fillMissingTaskInformation(tasks, tasksFromDB) {
```

Comment:
@borkarsaish65 Please add function doc, and what will this function do?

Reply:
done